### ISSUE TITLE
Zed's "Application Kind" is "iOS"

### ISSUE THREAD

**Issue by rowillia** [zed-industries/zed#15493]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Repro:

1. Open up "System Information.app"
2. Go to "Software > Applications" in the right hand side, scroll down to "Zed"

This isn't a _huge_ issue, so if it's more trouble than it's worth to fix (e.g. breaks auto-update) don't worry too much about it, but it did make our security team raise an eyebrow.

<img width="1235" alt="Screenshot 2024-07-30 at 12 45 01 PM" src="https://github.com/user-attachments/assets/9a5707fb-f798-4764-83f1-8ec1cfab8244">

### Environment

OSX

### If applicable, add mockups / screenshots to help explain present your vision of the feature

N/A

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by notpeter** [comment]
Looks like a bunch of other apps show the same thing.
<img width="918" alt="Screenshot 2024-07-30 at 13 23 26" src="https://github.com/user-attachments/assets/471c5814-ce7d-496a-82dd-39fb0d52ab2e">

**Comment by rowillia** [comment]
@notpeter Yeah, TBH I'm not sure what the right answer here is, I won't be offended if it's closed as "won't fix" but it might come up with other customers 🤷

**Comment by notpeter** [comment]
I'm open to ideas, but am not sure what actions would change this. Our pipeline is open source and clearly other folks (VLC, etc) are living with the same status quo.

I'm going to leave it open for the moment, but welcome PRs! 🙂

**Comment by notpeter** [comment]
I'm going to go ahead and close as unactionable.
If anyone has ideas or knows how to solve this happy to review PRs / discuss further.
